### ISSUE TITLE
Is the latest 7.4 version dsl available for communication with ES 7.10 server

### ISSUE THREAD

**Issue by qinghuan1998** [elastic/elasticsearch-dsl-py#1580]
I would like to know whether the existing latest version of dsl is universal to any version of ES 7.x, including the officially released version 7.17, thank you

**Comment by hnarayanan** [comment]
It works for my project.

**Comment by sethmlarson** [comment]
Yes, the major version is the only number that matters for determining compatibility between elasticsearch-dsl and the server.
